e2e-tests/package.json

```diff
@@ -4,13 +4,57 @@
   "private": true,
   "description": "End-to-end tests for Apex Language Server Extension",
   "scripts": {
-    "test:e2e": "playwright test",
-    "test:debug": "DEBUG_MODE=1 playwright test --debug",
-    "test:visual": "DEBUG_MODE=1 playwright test --ui",
+    "test:e2e": "wireit",
+    "test:debug": "wireit",
+    "test:visual": "wireit",
     "server": "node test-server.js",
     "clean": "echo 'No artifacts to clean in e2e-tests'",
     "clean:all": "npm run clean"
   },
+  "wireit": {
```

Comment:
just to confirm, this will cache (b/c `output` and `files` are present).  The other ones are marked `cache: false`.